Directory.Packages.props

```diff
@@ -0,0 +1,30 @@
+<Project>
+  <PropertyGroup>
+    <ManagePackageVersionsCentrally>true</ManagePackageVersionsCentrally>
```

Comment:
All that file and things around central package management feature are too verbose and honestly excessive for small projects. I won't bring anything regarding central package management in repo.

Reply:
I'm under the impression that MS is heading in that direction. That being said, I agree that it can feel as overhead for small projects (although I'm happy that I switched myself for some of my small projects).

I think the main issue with some of the more opinionated rules is that due to some crappy support by MS, it is less trivial to disable (or enable) rules in a .globalconfig or .editorconfig file. For rules about C# code this works perfectly.

Reply:
I'm fine disabling rules by `NoWarn` as I did here https://github.com/destructurama/attributed/pull/143.

Reply:
Having it in a `.globalconfig` file is way more readable: https://github.com/Qowaiv/Qowaiv/blob/master/.globalconfig

In a `<NoWarn>` tag, you can not easily see which rules are disabled.  I still have to take some action to report this to MS with a simple analyzer that makes it easy for them to reproduce the issue and hopefully fix it (or show me another way other than `<NoWarn>`).

Reply:
>  is way more readable

I don't want to speculate around what way is more readable or not. Experience varies.